src/utils/hydration.js

```diff
@@ -142,19 +157,75 @@ function injectRepeatedRows({ step, repeatedRowCount }) {
   };
 }
 
+const formatTrackValue = (parentDimension, value, type = 'col', isEmpty) => {
+  if (isAuto(value)) {
+    if (!isAuto(parentDimension)) {
```

Comment:
can we add quick comments explaining what each of these if statement branches are